.pre-commit-config.yaml

```diff
@@ -26,7 +26,8 @@ repos:
         # Formats Python, Pyi, and Jupyter files, excluding specified directories
         - id: ruff-format
           types_or: [ python, pyi, jupyter ]
-          exclude: (docs/)
+          exclude: (docs/|/tests/)
```

Comment:
Why? I don't see any reason why tests shouldn't be correctly formatted.

Reply:
Fixed.